etc/config-template.toml

```diff
@@ -56,6 +56,18 @@
 # max-tasks-per-worker-low = 2000
 # stack-size = "10MB"
 
+[http]
+## If HTTP service is enabled. If the value is true, the Prometheus can pull the metrics directly 
+## from it through the HTTP address. The reason we need it is Prometheus only support HTTP, 
+## but TiKV only supports gRPC and the C gRPC can't support dispatching different protocols.
```

Comment:
Maybe "The reason ......." can be removed, since it is obvious that this port is for Prometheus pulling metrics. The user doesn't care about whether or not Prometheus supports multiple protocols and whether or not our gRPC implementation supports multiplexing (which is implementation details). What really matters is that, what is the consequence of this configuration entry. For example, it's better to also state that, if the value is false, users must rely on metrics pushing (instead of pulling).